core/domain/fs_domain.py

```diff
@@ -201,6 +203,22 @@ def delete(self, filepath):
         except cloudstorage.NotFoundError:
             raise IOError('Image does not exist: %s' % filepath)
 
+    def copy2(self, source_assets_path, filepath):
```

Comment:
Why are we calling it copy? If this is just for GCS compatibility then call it copy() and add a comment above the cloudstorage.copy2 line to explain the use of copy2.

Reply:
Ah, sorry this was my mistake! I'll push a fix, soon!

Reply:
> Why are we calling it copy?

You mean "why are we calling it copy2", right?

Reply:
Done!